Utilities/CodeGeneration/EdFi.Ods.CodeGen/Program.cs

```diff
@@ -90,7 +92,10 @@ await _container.Resolve<IApplicationRunner>()
             {
                 var baseException = e.GetBaseException() ?? e;
 
-                _logger.Error(baseException.Message);
+                // Save the last exception in a static property to make it available for subsequent inspection.
+                LastException = e;
```

Comment:
Make sure we make the exception available to the calling code when we complete the call (from unit tests).